script/config-center/apollo/apollo-config.sh

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env bash
+# ----------------------------------------------------------------------------
+#  Copyright 2001-2006 The Apache Software Foundation.
```

Comment:
The copyright is not match with project, you can copy form any java file in head line directly.